recipe/meta.yaml

```diff
@@ -21,12 +21,13 @@ requirements:
   build:
     - pkg-config  # [not win]
     - python
-    - zeromq      # [not win or (win and py35)]
+    - zeromq      # [not win]
     - libsodium   # [not win]
   run:
     - python
-    - zeromq      # [not win or (win and py35)]
+    - zeromq      # [not win]
     - libsodium   # [not win]
+    - vs2015_runtime  # [win and py35]
```

Comment:
So Python 3.5 at conda-forge now [includes]( https://github.com/conda-forge/python-feedstock/blob/ef6fe0f7719ce4b973d406938f0cedab5cc4e67b/recipe/meta.yaml#L43 ) the VS runtime. This can be verified in the [64-bit package]( https://anaconda.org/conda-forge/python/files/modal/info/5796c67dfe3f5a0d64d9a278 ) and [32-bit package]( https://anaconda.org/conda-forge/python/files/modal/info/5796c634fe3f5a1cabcbe85b ). As a result, I think we can drop this.